src/Android/Android.csproj

```diff
@@ -202,7 +203,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\App\App.csproj">
-      <Project>{9F1742A7-7D03-4BB3-8FCD-41BC3002B00A}</Project>
+      <Project>{EE44C6A1-2A85-45FE-8D9B-BF1D5F88809C}</Project>
```

Comment:
VSwin and VSmac seem to argue endlessly about this value.  We've established VSwin as our gold standard  so this is just a reminder to revert this before finalizing the PR.  (fwiw Rider is polite enough to leave it alone)

Reply:
Ah, that is odd considering the `.sln` file actually has the App project listed with UUID, `EE44C6A1-2A85-45FE-8D9B-BF1D5F88809C`, which VSmac seems to be picking up and correcting in the project reference in the Android project. Not sure where VSwin is getting the UUID, `9F1742A7-7D03-4BB3-8FCD-41BC3002B00A` since it exists no where in the `.sln` file or the `App.csproj` file. very bizarre; but I'll undo it before PR-time.